README.md

```diff
@@ -20,22 +20,18 @@ yarn start
 
 ## Deploying the UI
 
-1. Change `index.html` to point to `dist/index.js` (instead of `example.js`) and upload to s3 bucket (or other file hosting service).
-2. Upload `example.css`.
-3. Build the js/css bundle (which will appear in the `dist/` directory) and upload.
-
-Here's an example set of commands using the AWS CLI:
-```bash
-# The first two steps only need to be performed the first time.
-aws s3 cp index.html s3://my-bucket --acl public-read
-aws s3 cp example.css s3://my-bucket --acl public-read
-# Anytime the otp-react-redux code changes, you'll need to run the following steps:
-yarn build
-aws s3 cp --recursive dist/ s3://my-bucket/dist --acl public-read
-```
-
-You can then view your site at: https://my-bucket.s3.amazonaws.com/index.html
-
+1. Build the js/css bundle by running `yarn build`. The build will appear in the `dist/` directory).
+2. Modify the `index.html` to point to `dist/index.js` (instead of `example.js`).
+3. Upload the following files to wherever you're deploying the UI:
+  - `index.html` (modified to point to `dist/index.js`)
+  - `example.css`
+  - `dist/`
+    - `index.js`
+    - `index.js.map`
+    - `index.css`
+    - `index.css.map`
+
+Note: only contents produced during build in the `dist/` directory are likely to change over time (the `index.html` and `example.css` files contain minimal code), so subsequent deployments will typically only need to replace the `dist/` contents.
```

Comment:
The instructions are good, Landon.
That said, not sure what to do with call-config.js 

When build was under trimet-mod, this is what I was doing: yarn build --env.JS_CONFIG=$BUILDER/config/call-config.js --env.YAML_CONFIG=$BUILDER/config/config.yml

Reply:
BTW, I'm also not clear on the whole fieldtrip module.  And also the trinet auth workflow (not that that needs to go here).

Reply:
BTW, Landon explained call-config.js (no purpose with OTP-RR ... was a remnant from trimet-MOD project).  Also have the call app running in our test environ, complete with trinet auth and connections to the call db.

Reply:
@fpurcell, I'll add a brief section pointing people to deployment instructions for trimet-mod-otp. I think that might be helpful for future users of this product.